packages/asset-service/src/service/AssetService.ts

```diff
@@ -21,22 +22,33 @@ export class AssetService {
     return this.assets
   }
 
-  async description(assetId: AssetId): Promise<DescriptionData> {
-    const descriptions: Record<string, string> = assetsDescriptions
-    const description = descriptions[assetId]
-
+  async description(assetId: AssetId, locale = 'en'): Promise<DescriptionData> {
+    const localeDescriptions = assetsDescriptions[locale]
     // Return overridden asset description if it exists and add isTrusted for description links
-    if (description) return { description, isTrusted: true }
+    if (localeDescriptions[assetId] || assetsDescriptions.en[assetId]) {
+      const polyglot = new Polyglot({
+        phrases: localeDescriptions,
+        allowMissing: true,
+        onMissingKey: (key) => assetsDescriptions.en[key] // fallback to English overriden description, which should always be added as a base translation
```

Comment:
does this not negate the need for all of the non-english files with the same translations as english?

Reply:
as discussed they're effective placeholders that lower risk for when translations are available, LGTM

Reply:
We don't need them yet, but this is provided with all translations ready to be modified so that globalization can provide them after we merge it, making things smoother 🥥